packages/report-server/src/routes/helpers.ts

```diff
@@ -3,8 +3,10 @@
  * Copyright (c) 2017 - 2021 Beyond Essential Systems Pty Ltd
  */
 
-import type { EntityApi } from '@tupaia/api-client';
-import type { AccessPolicy } from '@tupaia/access-policy';
+import { AccessPolicy } from '@tupaia/access-policy';
+import { MicroServiceRequestContext } from '@tupaia/server-boilerplate';
+
+type EntityApi = MicroServiceRequestContext['services']['entity'];
```

Comment:
I don't mind this change, but am also not opposed to the previously behaviour of directly importing `EntityApi` from `api-client`? I think some degree of awareness that services are instances of various `api-client`s is fine throughout the system?

Reply:
To do that, I'd typically need to declare `@tupaia/api-client` as a dependency of this package.... I changed this to fix an existing EsLint error.

What I could do is expose some `@tupaia/api-client` types to `@tupaia/server-boilerplate` consumers through the latter... I will try that

Reply:
Ah, so typescript has something that works really nicely for that scenario. There's an `import type` command, which you can use to import a type from somewhere without declaring that package as a dependency 👍 Since the typescript stuff is all just compile time all `import type` statements get removed when code is built!

Reply:
Ah that's really nice! Turns out the `import type` syntax was used in the first place, and the error was a bit different (`EntityApi` is not exported in that package)